lib/functions/index.js

```diff
@@ -1131,7 +1131,8 @@ exports.match = function match(pattern, val){
   [].slice.call(arguments).forEach(function(expr){
     expr = utils.unwrap(expr);
     if (!expr.nodes.length) return;
-    console.log('\033[90minspect:\033[0m %s', expr.toString().replace(/^\(|\)$/g, ''));
+    var _033 = parseInt('033', 8);
+    console.log(_033 + '[90minspect:' + _033 + '[0m %s', expr.toString().replace(/^\(|\)$/g, ''));
```

Comment:
There should be `\u001b` instead of `\033`. See http://etutorials.org/Programming/actionscript/Appendix+A.+Unicode+Escape+Sequences+for+Latin+1+Characters/